src/nativeAssetManager.js

```diff
@@ -227,12 +271,43 @@ export function newNativeAssetManager(win) {
     }
 
     if (data.message === 'assetResponse') {
-      const body = win.document.body.innerHTML;
-      const newHtml = replace(body, data);
+      const body = win.document.body.innerHTML,flag = (typeof win.pbNativeData !== 'undefined');
 
-      win.document.body.innerHTML = newHtml;
-      callback && callback();
-      win.removeEventListener('message', replaceAssets);
+      if((data.hasOwnProperty('rendererUrl') && data.rendererUrl) || (flag && win.pbNativeData.hasOwnProperty('rendererUrl'))) {
+        if(win.renderAd){
```

Comment:
@jaiminpanchal27 When the rendererUrl is loaded, is the expected output that a renderAd function is placed on the win. 

If you could review this use case a little more in depth I would appreciate it since it was spec'd by you.